src/main/java/com/ericsson/ei/jsonmerge/MergePrepare.java

```diff
@@ -135,8 +135,10 @@ public String getMergePathFromArrayMergeRules(String originObject, String mergeR
         log.debug(" originObject is : " + originObject);
         try {
             JSONArray ruleJSONArray = new JSONArray(mergeRule);
-            String firstRule = ruleJSONArray.getString(0);
-            String secondRule = ruleJSONArray.getString(1);
+            String firstRule = "";
+            String secondRule = "";            
+            firstRule = ruleJSONArray.get(0).toString();
+            secondRule = ruleJSONArray.get(1).toString();
```

Comment:
What is the reason that this isn't done above at the init of the variables? It won't be that long line right?